aggregator/forms.py

```diff
@@ -47,3 +49,85 @@ def clean_feed_url(self):
                 "here automatically."
             )
         return feed_url
+
+
+class LocalDjangoCommunityModelForm(forms.ModelForm):
+    name = forms.CharField(
+        max_length=250,
```

Comment:
In models.py this field is defined as `name = models.CharField(max_length=120)` so max_len should be 120